cli/cmd/plugin/ui/web/tanzu-ui/src/components/aws/cluster-settings-step/ClusterSettings.tsx

```diff
@@ -0,0 +1,167 @@
+// React imports
```

Comment:
I'd prefer "management" in both the cluster-settings-step name and the ClusterSettings.tsx name

Reply:
renamed